scripts/train/README.md

```diff
@@ -17,7 +17,10 @@ You can read more about the benefits of using mosaicml-streaming [here](https://
 NOTE: If you only want to profile these LLMs, we recommend that you **download and prepare the `train_small` and `val_small` splits**,
 and skip the full `train` and `val` splits. You'll just need to replace `split: train` with `split: train_small`
 and `split: val` with `split: val_small` in your run YAML's dataloader config.
-You can also accomplish this in your CLI command like so: `composer train.py ... train_loader.dataset.split=train_small eval_loader.dataset.split=val_small`
+You can also accomplish this in your CLI command like so: 
```

Comment:
We don't really introduce composer or the full correct command here. Not sure how self-contained this should all be